backend/substrapp/tasks/tasks.py

```diff
@@ -583,13 +581,8 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         channel_name, tuple_type, subtuple, compute_plan_key = self.split_args(args)
 
         try:
-            error_code = compute_error_code(exc)
-            # Do not show traceback if it's a container error as we already see them in
-            # container log
-            type_exc = type(exc)
-            type_value = str(type_exc).split("'")[1]
-            logger.error(f'Failed compute task: {tuple_type} {subtuple["key"]} {error_code} - {type_value}')
-            log_fail_tuple(channel_name, tuple_type, subtuple['key'], error_code)
+            logger.error(f'Failed compute task: {tuple_type} {subtuple["key"]}')
+            log_fail_tuple(channel_name, tuple_type, subtuple['key'], "")
```

Comment:
As this looks like the only usage of log_fail_tuple() in the codebase could we change the signature to `log_fail_tuple(channel_name, tuple_type, tuple_key)` in `backend/substrapp/ledger/api.py` ?

Reply:
`log_fail_tuple` expect a string (even empty), I'd prefer to keep the behavior in case we need to add log when tuple fails :)